### ISSUE TITLE
Win10 Random Crash

### ISSUE THREAD

**Issue by hmnd** [Pulover/PuloversMacroCreator#34]
The program crashes my Win10 laptop randomly. It also makes everything really slow until I click the Close Window button on the Windows troubleshooter dialog box.
This is the only log I could find:

``` 1,"fusion","GAC",0
1,"WinRT","NotApp",1
3,"System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089","C:\WINDOWS\assembly\NativeImages_v4.0.30319_64\System\ee1c3ea3e0049df6d53b35e21c2247e4\System.ni.dll",0
3,"System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a","C:\WINDOWS\assembly\NativeImages_v4.0.30319_64\System.Drawing\3ac58fb7cb8d849d0f85f2435e1a0066\System.Drawing.ni.dll",0
3,"System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089","C:\WINDOWS\assembly\NativeImages_v4.0.30319_64\System.Core\46ebe2b057622677cfc3fd04f47081c0\System.Core.ni.dll",0
3,"System.Configuration, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a","C:\WINDOWS\assembly\NativeImages_v4.0.30319_64\System.Configuration\a7f47973b917fdac945dd35f56b99eaf\System.Configuration.ni.dll",0
3,"System.Xml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089","C:\WINDOWS\assembly\NativeImages_v4.0.30319_64\System.Xml\42142f2f161821b78f345ecf759fa095\System.Xml.ni.dll",0
3,"System.ServiceModel, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089","C:\WINDOWS\assembly\NativeImages_v4.0.30319_64\System.ServiceModel\9d0f658faeab6352336ec6d83411ed47\System.ServiceModel.ni.dll",0
3,"System.Runtime.Serialization, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089","C:\WINDOWS\assembly\NativeImages_v4.0.30319_64\System.Runteb92aa12#\d55e112b638ed61c3240dd7e16670fd1\System.Runtime.Serialization.ni.dll",0
```

Thanks!

**Comment by centrual** [comment]
Yes, it's crashing randomly in Windows 10. I tested:
1) x32 - use %appData% as config directory
2) x32 - use program directory as config directory ( must run as admin otherwise can't read latest config )
3) x64 - use %appData% as config directory
4) x64 - use program directory as config directory ( must run as admin otherwise can't read latest config )

**Comment by Xeddicus** [comment]
I'm not crashing, but it just basically locks up for me/is horribly slow on Win 10 X64.

**Comment by dbline** [comment]
Same problem here.  Looks like this project has been abandoned now, no updates since 2016.  Time to move on I guess.

**Comment by asas-dev** [comment]
Any news on this issue or recommendations for which direction to move on?

**Comment by roshad** [comment]
64 ver lose response often, 32 ver OK

**Comment by Pulover** [comment]
Fixed in v5.1.2. [Read more](https://www.macrocreator.com/2020/09/14/version-update-5-1-2/).
